pom.xml

```diff
@@ -97,7 +99,7 @@
         <artifactId>maven-hpi-plugin</artifactId>
         <extensions>true</extensions>
         <configuration>
-          <compatibleSinceVersion>2.0.0</compatibleSinceVersion>
+          <compatibleSinceVersion>${jenkins.version}</compatibleSinceVersion>
```

Comment:
It's our plugin version (and compatibility between them), not the Jenkins one.

Reply:
Ah, ok.